client-js/app/elements/labrad-grapher.html

```diff
@@ -64,17 +64,28 @@
       font-size: 24px;
       line-height: 45px;
     }
-    #params thead tr {
-      background-color: #7744AA;
+    #params thead th {
+      background-color: #3f51b5;
```

Comment:
I liked having a different color here; doesn't have to be the same purple, but something other than blue is nice to give some visual variety IMO